pimd/pim_ssmpingd.c

```diff
@@ -273,15 +279,17 @@ static int ssmpingd_read_msg(struct ssmpingd_sock *ss)
 		char source_str[INET_ADDRSTRLEN];
 		char from_str[INET_ADDRSTRLEN];
 		char to_str[INET_ADDRSTRLEN];
+
 		pim_inet4_dump("<src?>", ss->source_addr, source_str,
 			       sizeof(source_str));
-		pim_inet4_dump("<from?>", from.sin_addr, from_str,
+		pim_inet4_dump("<from?>", from_addr->sin_addr, from_str,
 			       sizeof(from_str));
-		pim_inet4_dump("<to?>", to.sin_addr, to_str, sizeof(to_str));
+		pim_inet4_dump("<to?>", to_addr->sin_addr, to_str,
+			       sizeof(to_str));
 		zlog_warn(
 			"%s: bad ssmping type=%d from %s,%d to %s,%d on interface %s ifindex=%d fd=%d src=%s",
```

Comment:
use `%pSUp` to print `struct sockaddr_in`, remove formatting calls & variables above (`p` option = includes port)

Reply:
Ack